xdsl/ir/core.py

```diff
@@ -755,7 +755,9 @@ def walk_reverse(self) -> Iterator[Operation]:
             yield from region.walk_reverse()
         yield self
 
-    def verify(self, verify_nested_ops: bool = True) -> None:
+    def verify(
+        self, verify_nested_ops: bool = True, allow_unregistered: bool = False
```

Comment:
Could we always have `allow_unregistered` to `True`?
I'm not sure why we would disallow it at least.
I'm going to add a PR anyway that sets "allow_unregistered" in the `MLContext`, so technically we can disallow unregistered operations entirely from the context, rather than in the parser only.

Reply:
Yeah, I was trying to preserve the meaning of the operation as it was when I didn't introduce the argument.
I can change the default value.